pom.xml

```diff
@@ -19,12 +19,6 @@
     </properties>
 
     <dependencies>
-        <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.12</version>
-            <scope>test</scope>
-        </dependency>
     </dependencies>
```

Comment:
you don't need to edit these files. If you encounter some problems, don't hesitate to ask in the chat